definitions/w3c-customelements/index.d.ts

```diff
@@ -0,0 +1,51 @@
+// Type definitions for Custom Elements 1.0
+// Project: https://w3c.github.io/webcomponents/spec/custom/
+// Definitions by: vvakame <https://github.com/vvakame>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export interface HasAttributes {
+    observedAttributes: string[];
+}
+
+export interface OnConnectedCallback {
+    connectedCallback(): void;
+}
+
+export interface OnDisconnectedCallback {
+    disconnectedCallback(): void;
+}
+
+export interface OnAdoptedCallback {
+    adoptedCallback(oldDocument: any, newDocument: any): void;
+}
+
+export interface OnAttributeChangedCallback {
+    attributeChangedCallback(name: string, oldValue: any, newValue: any): void;
+}
+
+declare global {
```

Comment:
It works with the current code. Is this not okay?
NOTE: https://github.com/Microsoft/TypeScript/wiki/What%27s-new-in-TypeScript#augmenting-globalmodule-scope-from-modules